functions/src/routes/express_portal.ts

```diff
@@ -34,5 +35,38 @@ app_portal.get("/auth0/applications", get_active_applications);
 app_portal.get("/auth0/discord", get_user_metadata);
 app_portal.post("/auth0/verify-discord", verify_in_acm_server);
 
+// Create Vanity Link
+app_portal.post(
+  "/auth0/vanity/create",
+  [
```

Comment:
Is there a way to shift the middleware to the [config file](https://github.com/acmutd/portal-backend/blob/dev/functions/src/express_configs/express_portal.ts) instead of having it be in the routes file? Mostly for readability purposes.